db/migrate/20221202115419_add_user_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] Taking into consideration the given ERD, the reference name should be `author` instead of `user`, hence creating a migration query should be similar to:

```ruby
...
  def change
    add_reference :posts, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...